feat(python): Add a workaround to GCP docs (#8486)
getsentry/sentry-docs: src/platforms/python/integrations/gcp-functions/index.mdx

The GCP integration currently supports only the Python `3.7` runtime environment. If you try to run it with a newer version (such as `3.9`), an error message will be displayed (with debug mode on).

> Even though they're not directly supported, you can still get some Sentry support on newer Python runtimes by directly enabling [integrations for supported frameworks](/platforms/python/integrations) and/or using the generic [serverless integration](/platforms/python/integrations/serverless/).

Reviewer: ```suggestion
Limited Sentry support is available on newer Python runtimes by directly enabling [integrations for supported frameworks](/platforms/python/integrations) and/or using the generic [serverless integration](/platforms/python/integrations/serverless/).
```

Author: Looks good, thanks! Applied.
